manifests/crd.yaml

```diff
@@ -1,68 +1,113 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    "api-approved.kubernetes.io": "https://github.com/kubernetes/enhancements/pull/1870"
```

Comment:
as I remember at the time of initial commit, about a year ago this field: api-approved.kubernetes.io was mandatory. For controlller-gen it's not, right? Where we can see a reference?

Reply:
Good point, addressed in the newest version of the pr. I added a kubectl command to add the annotation